requirements/pytest.txt

```diff
@@ -1,5 +1,5 @@
 # consciously added packages
--r base.txt
+-r prod.txt
```

Comment:
Well, I didn't want to insert pg bindings into every file. The reasoning is that requirements is not configuration, so production will not be affected if its dependencies are used by tests.